plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/FeatureDevSessionContext.kt

```diff
@@ -66,9 +66,10 @@ class FeatureDevSessionContext(val project: Project, val maxProjectSizeBytes: Lo
         "dist/"
     ).map { Regex(it) }
 
-    private var _projectRoot = project.guessProjectDir() ?: error("Cannot guess base directory for project ${project.name}")
+    val projectRoot = project.guessProjectDir() ?: error("Cannot guess base directory for project ${project.name}")
+    private var _currentRoot = projectRoot
```

Comment:
[question]: I wonder if this is a well-descriptive name or not. Maybe `selectedFolder` would be better? Since this isn't a root folder anymore. 

Also, it would help to have some comments in the code to describe what is projectRoot for and what is currentRoot for.

Reply:
I see, yes, it could be made more similar even to what the feature is about, i.e. `selectedSourceFolder`, and I'll add comments as well, sure.